app/scripts/views/permissions.js

```diff
@@ -0,0 +1,135 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+define([
+  'cocktail',
+  'underscore',
+  'views/form',
+  'views/base',
+  'stache!templates/permissions',
+  'lib/promise',
+  'views/mixins/back-mixin',
+  'views/mixins/service-mixin'
+],
+function (Cocktail, _, FormView, BaseView, Template, p,
+    BackMixin, ServiceMixin) {
+
+  var View = FormView.extend({
+    template: Template,
+    className: 'permissions',
+
+    initialize: function (options) {
+      // Account data is passed in from sign up and sign in flows.
+      var data = this.ephemeralData();
+      this._account = data && this.user.initAccount(data.account);
+
+      this.type = options.type;
```

Comment:
Can we determine that from the flow of screen events (i.e. if they came from sign-in or sign-up)? The process for adding metrics still seems rather ad-hoc... not sure all the questions we'd want to answer, or if @ryanfeeley should be looped in, ect.